pontus/amazon_s3_file_validator.py

```diff
@@ -50,14 +51,19 @@ class AmazonS3FileValidator(object):
     :param validators:
         List of validators. A validator can either be an instance of a class
         inheriting :class:`BaseValidator` or a callable
-        function that takes Boto S3 Key instance as a parameter.
+        function that takes Boto S3 Object instance as a parameter.
 
     """
     def __init__(self, key_name, bucket, validators=[]):
         self.errors = []
-        self.key = bucket.get_key(key_name)
-        if not self.key:
-            raise FileNotFoundError(key=key_name)
+        self.key = bucket.Object(key_name)
```

Comment:
Should we rename `key` to `object` or `obj`? The terminology change seems to be `Key` -> `Object`, `key_name` -> `key`.